spec/models/application_record_spec.rb

```diff
@@ -206,56 +238,74 @@
       # technically. gender_eeo_data uses it.
       call
     end
+    it 'calls AR#in_department to filter application records' do
+      relation = ApplicationRecord.all
+      expect(ApplicationRecord).to receive(:in_department)
+        .with(@department.id)
+        .and_return(relation).at_least(:once)
+      # needs to return an activerecord relation. Also
```

Comment:
This comment could be worded more clearly, and should also go in the other places where we mock in `ApplicationRecord.none`.